build.assets/windows/build.ps1

```diff
@@ -444,6 +444,44 @@ function Build-Connect {
     Write-Host $("Built Teleport Connect in {0:g}" -f $CommandDuration)
 }
 
+function Write-Version-Objects {
+    [CmdlesBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [string] $TeleportSourceDirectory,
+        [Parameter(Mandatory)]
+        [string] $TeleportVersion,
+    )
+    Write-Host "Generating version info files for tsh.exe and tctl.exe..."
+
+    # install go-winres (v0.3.3)
+    go install github.com/tc-hib/go-winres@d743268d7ea168077ddd443c4240562d4f5e8c3e
+
+    $Year = (Get-Date).Year
+
+    # generate tsh version info
+    go-winres simply --no-suffix --arch amd64 `
+        --file-description "Teleport tsh command-line client" `
+        --original-filename tsh.exe `
+        --copyright "Copyright (C) $Year Gravitational, Inc." `
+        --icon "$TeleportSourceDirectory\e\windowsauth\installer\teleport.ico"
+        --product-name Teleport `
+        --product-version $TeleportVersion `
+        --file-version $TeleportVersion `
+        --out "$TeleportSourceDirectory\tool\tsh\resource.syso"
```

Comment:
Where is this file used? Does some other tool pull it in?

Reply:
Yes, `go build` uses it.